website/content/docs/get-started/operations-qs.mdx

```diff
@@ -0,0 +1,190 @@
+
+1. Open a new terminal and export environment variables to interact with the
+   Vault server.
+
+   ```shell-session
+   $ export VAULT_ADDR='http://127.0.0.1:8200'
+   ```
+
+1. Log into the Vault server using the root token.
+
+   ```shell-session
+   $ vault login dev-only-token
+   ```
+
+   You are now authenticated to Vault with the root token.
+
+## Enable auth methods
+
+Vault supports multiple [authentication methods](/vault/docs/auth). These
+plugins authenticate both human and machine based workloads to Vault. In this
+quick start, you will use the `userpass` auth method.
+
+1. Enable the `userpass` auth method. This method allows users to authenticate to
+   Vault.
+
+   ```shell-session
+   $ vault auth enable userpass
+   ```
+
+   The `userpass` auth method is enabled.
+
+1. To authenticate users with Vault, you must create a user and password.
+
+   ```shell-session
+   $ vault write auth/userpass/users/opsuser \
+       password=p@ssw0rd
+   ```
+
+   The user `opsuser` is created. The password is `p@ssw0rd`.
+
+1. (Optional) Enable the `approle` auth method. This is a general purpose plugin that
+   allows machines or apps to authenticate to Vault.
+
+   ```shell-session
+   $ vault auth enable approle
+   ```
+
+   The `approle` auth method is enabled.
+
+1. (Optional) To authenticate workloads with Vault, you must create a role.
+
+   ```shell-session
+   $ vault write auth/approle/role/my-app-role \
+       secret_id_ttl=10m \
+       token_ttl=20m \
+       token_max_ttl=30m
+   ```
+
+   The role `my-app-role` is created. The role has a secret ID TTL of 10 minutes, a
+   token TTL of 20 minutes, and a token max TTL of 30 minutes.
+
+## Enable secrets engines
+
+Vault supports multiple [secrets engines](/vault/docs/secrets). These
+plugins support various use cases like storing static secrets, dynamic secrets,
+and managing PKI certificates.
+
+Enable the `kv` secrets engine. This engine stores arbitrary secrets.
+
+```shell-session
+$ vault secrets enable -version=2 kv
+```
+
+The `kv` secrets engine is enabled.
+
+## Store a secret
+
+Vault securely stores secrets static for a configured `kv` secrets engine.
+
+Write a secret to the `kv` secrets engine.
+
+```shell-session
+$ vault kv put kv/creds username=opsuser password=p@ssw0rd
+```
+
+The `kv put` command writes the secret to the `kv` secrets engine. In this example, two
+secrets are written: The `username` key with a value of `opsuser` and the key
+`password` with a value of `p@ssw0rd`.
+
+## Protect secrets
+
+In the previous step, you wrote a password to the `kv` secrets engine using the
+root token. Using the root token, and root policy, is not recommended for production environments.
+
+Vault policies allow you to control access to secrets, and which operations are allowed.
+
+1. Write a policy that permits access to the `kv` secrets engine.
+
+   ```shell-session
+   $ vault policy write kv-access-policy - << EOF
+   path "kv/creds" {
```

Comment:
```suggestion
   path "shared/data/kv/creds/*" {
```

Not 100% about this, but the example as written doesn't match the syntax we use for the `kv` plugin setup guide.